### PR TITLE
Only create alarms in PROD

### PR DESCRIPTION
**Pull request by rupertbates**
## What are you doing in this PR?
We don't want to create alarms in code for support-frontend. Previously we had a boolean which meant they were inactive, but there's no point in creating them at all as they just clutter things up, so this PR ensures that alarms are only created in PROD.

### CODE REVIEW

File: cdk/lib/frontend.ts [hunk 1]
```diff
@@ -124,15 +135,8 @@ export class Frontend extends GuStack {
...
-        http5xxAlarm: {
-          alarmName: alarmName("support-frontend is returning 5XX errors"),
-          alarmDescription: alarmDescription(
-            "Some or all actions on support website are failing"
-          ),
-          actionsEnabled: shouldEnableAlarms,
-          tolerated5xxPercentage: 5,
-        },
-        unhealthyInstancesAlarm: true,
+        http5xxAlarm: http5xxAlarm,
```
Comment (andrewHEguardian) [inline cdk/lib/frontend.ts:138]: What happens here when the `http5xxAlarm` has value false?

**Review by andrewHEguardian — APPROVED** [review]
Nice 👍

File: cdk/lib/frontend.ts (hunk 1 above)
Comment (rupertbates) [reply cdk/lib/frontend.ts:138]: It just doesn't create one.

**Comment by johnduffell** [comment]
> there's no point in creating them at all as they just clutter things up

...and make it easier to test in CODE

**Pull request merged**